.github/workflows/tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Run PR Tests
+
+on:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Should we use setup-purescript or npm install?
+      # - name: Set up PureScript toolchain
+      #   uses: purescript-contrib/setup-purescript@main
```

Comment:
I think using `setup-purescript` and overriding the values there to `v0.13.x` should be done.

But yeah, we need to update this repo to `v0.14.x`